frontend/src/helpers/utils.ts

```diff
@@ -84,20 +84,21 @@ export function parseJSONsafe(presumedJSON: string): any | null {
 export function formatDateToIsoString(dateToIsoString: Date) {
   return format(dateToIsoString, "yyyy-MM-dd");
 }
-export function calculateItemSumPrice(
-  arrayItems: Pick<OfferItem, "amount" | "price">[],
-  date?: string,
-) {
+export function calculateItemSumPrice(arrayItems: (OfferItem | OfferItemCreate)[], date?: string) {
```

Comment:
I only saw this now. Why is the date optional?

Reply:
Function `getVatRate` supports optional value of date. Therefore, `calculateItemSumPrice` also supports the same scenario.